packages/@webex/media-helpers/src/webrtc-core.ts

```diff
@@ -29,18 +31,22 @@ export type ServerMuteReason =
   | 'localUnmuteRequired'; // server forced the client to be unmuted
 
 // these events are in addition to WCME events. This will be properly typed once webrtc-core event types inheritance is fixed
-export enum LocalMicrophoneTrackEvents {
+export enum LocalMicrophoneStreamEventNames {
```

Comment:
I made this change in my example branch because that's what we were doing in webrtc-core. The event names are defined as a separate variable (e.g. `LocalStreamEventNames`) and the actual events are defined within the class. Here, the actual event for `LocalMicrophoneStreamEventNames.ServerMuted`, for example, is defined in lines 46-49.